darshan-util/pydarshan/darshan/experimental/plots/plot_io_cost.py

```diff
@@ -133,6 +128,8 @@ def plot_io_cost(report: darshan.DarshanReport, mod_key: str) -> Any:
     ax_norm.set_ylim(0, 100)
     # convert the normalized axis y labels to percentages
     ax_norm.yaxis.set_major_formatter(mtick.PercentFormatter())
+    if runtime > 1e4:
+        ax_raw.yaxis.set_major_formatter(mtick.StrMethodFormatter('{x:.0e}'))
```

Comment:
I think this change to scentific notation probably makes sense, though I'd generally suggest separating fixes into smaller bits when possible (and I don't think I saw the scientific notation explicitly requested yet, so probably could have been delayed for a smaller diff, etc.).

Reply:
Actually, I don't like the scentific notation yet--look at the plots above--the rounding of the decimal places leads to deceptive apparent spacing AND identical values plotted for neighboring ticks, which to me just doesn't make sense.

Reply:
Yeah, I'll have to take a stab at it again in a different PR I think. For now I've stripped the scientific notation changes out of this PR.